tutorials/intro/plot_object_annotations.py

```diff
@@ -1,245 +1,234 @@
 #
-# Concatenate
-# ~~~~~~~~~~~
+# Generally speaking, both the Events and :class:`~mne.Annotations` data
+# structures serve the same purpose: they provide a mapping between times
+# during an EEG/MEG recording and a description of what happened at those
+# times. In other words, they associate a *when* with a *what*. The main
+# differences are:
 #
-# It is possible to concatenate two annotations with the + operator (just like
-# lists) if both share the same ``orig_time``
-
-annot = mne.Annotations(onset=[10], duration=[0.5],
-                        description=['foobar'],
-                        orig_time=orig_time)
-annot = annot_orig + annot  # concatenation
-print(annot)
-
-###############################################################################
-# Iterating, Indexing and Slicing :class:`mne.Annotations`
-# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# 1. the Events data structure represents the *when* in terms of samples,
+#    whereas the :class:`~mne.Annotations` data structure represents the *when*
+#    in seconds.
+# 2. the Events data structure represents the *what* as an integer "Event ID"
+#    code, whereas the :class:`~mne.Annotations` data structure represents the
+#    *what* as a string.
+# 3. Events in an Event array do not have a duration (though it is possible to
+#    represent duration with pairs of onset/offset events within an Events
+#    array), whereas each element of an :class:`~mne.Annotations` object
+#    necessarily includes a duration (though the duration can be zero if an
+#    instantaneous event is desired).
+# 4. Events are stored as an ordinary :class:`NumPy array <numpy.ndarray>`,
+#    whereas :class:`~mne.Annotations` is a :class:`list`-like class defined in
+#    MNE-Python.
 #
-# :class:`~mne.Annotations` supports iterating, indexing and slicing.
-# Iterating over :class:`~mne.Annotations` and indexing with an integer returns
-# a dictionary. While slicing returns a new :class:`~mne.Annotations` instance.
 #
-# See the following examples and usages:
-
-# difference between indexing and slicing a single element
-print(annot[0])  # indexing
-print(annot[:1])  # slicing
-
-###############################################################################
-# How about iterations?
-
-for key, val in annot[0].items():  # iterate on one element which is dictionary
-    print(key, val)
+# What is a STIM channel?
+# ^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Many EEG and MEG systems incorporate one or more recording channels that do
+# not receive signals from a sensor; instead, those channels record voltages
+# (usually short, rectangular DC pulses of fixed magnitudes sent from the
+# experiment-controlling computer) that are time-locked to experimental events,
+# such as the onset of a stimulus or a button-press response by the subject. In
+# other cases, these pulses may not be strictly time-locked to an experimental
+# event, but instead may occur in between trials to indicate the type of
```

Comment:
sometimes you use the term "trial" and sometimes "epoch". I think it's better to stick to one. Unless there is a clear explanation somewhere that these two terms are equivalent. It might be worth explaining this somewhere anyway because "trial" is the terminology used in other software like FieldTrip

Reply:
I disagree that these two terms are equivalent (though some may use them as such). I try to use "trial" when talking about the experiment/experimental design and "epoch" when talking about extracted spans of data (which are usually trial-related but not necessarily, viz. resting state analysis).  I think I'm being consistent with that distinction here, but please tell me if (1) you notice somewhere else where I'm being inconsistent, or (2) you think that's not the right way to use the terminology.  

That said, I'd be fine with noting (in an epoching tutorial, not here) that what we call "epochs" are sometimes called "trials" in other software.

Reply:
I like @drammock distinction between the two. And this distinction of terms can be explained in the epoching tutorial. 

once said that, 

Do trials deserve an entry in the glossary? Could we just point to epochs in the glossary and make more clear that they are synonymous. Something like `:term: trial <epoch>` and then have a description like:

```rst
epochs
Also referred as **trails** when discussing experimental design, `Epochs <mne.Epochs>` are
chunks of data extracted from raw continuous data. See The Epochs data structure: epoched data for a narrative overview.
```

Reply:
That is basically what the epochs entry in the glossary already says:

> `Epochs are chunks of data extracted from raw continuous data. Typically, they correspond to the trials of an experimental design. See :class:`Epochs` for the API of the corresponding object class, and :ref:`tut-epochs-class` for a narrative overview.`

I don't like the idea of linking the word "trials" to the glossary entry for "epochs"... it might be confusing or seem broken to some users.  But I'll expand the glossary epochs entry a little bit to make it more clear that "epochs" = "trials" when comparing MNE-Python to some other softwares.